internal/storage/fake_storage_util.go

```diff
@@ -26,6 +26,23 @@ const TestObjectName string = "gcsfuse/default.txt"
 const ContentInTestObject string = "Hello GCSFuse!!!"
 const TestObjectGeneration int64 = 780
 
+type FakeStorageServer interface {
+	CreateStorageHandle() (storageHandleObj StorageHandle)
+
+	ShutDown()
+}
+
+type fakeStorage struct {
+	fakeStorageServer *fakestorage.Server
+}
+
+func NewFakeStorage(f *fakestorage.Server) FakeStorageServer {
```

Comment:
NewFakeStorage should create new fakestorage.server and return, instead of taking fakestorage.server as argument

Reply:
fixed